### ISSUE TITLE
Incorrect errors on AWS::SecurityHub::Hub configuration.

### ISSUE THREAD

**Issue by ps-spb** [aws-cloudformation/cfn-lint#2791]
### CloudFormation Lint Version

cfn-lint 0.77.10

### What operating system are you using?

MacOS

### Describe the bug

The following will throw up Invalid Properties in cfn-lint:

```  
SecurityHub:
    Type: AWS::SecurityHub::Hub
    Properties:
      # these errors aren't actual errors...
      AutoEnableControls: true
      ControlFindingGenerator: 'SECURITY_CONTROL'
      EnableDefaultStandards: true
```

The template is happily ingested via CloudFormation.

The above example aligns with the documentation for the item at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-securityhub-hub.html

### Expected behavior

No errors / issues are reported.

### Reproduction template

```  
SecurityHub:
    Type: AWS::SecurityHub::Hub
    Properties:
      # these errors aren't actual errors...
      AutoEnableControls: true
      ControlFindingGenerator: 'SECURITY_CONTROL'
      EnableDefaultStandards: true
```

**Comment by kddejong** [comment]
Included in the most recent spec release and accepted into the main branch with this PR.  https://github.com/aws-cloudformation/cfn-lint/pull/2772/files#diff-a5a6f378b19287fd419ef6c9edf44d4493163add0a727420af026573a02f165a

It will be included on the next release for now you can run `cfn-lint --update-specs`

**Comment by ps-spb** [comment]
Thanks!

**Comment by kddejong** [comment]
This is fix in v0.78.1
